.github/workflows/ci.yml

```diff
@@ -0,0 +1,97 @@
+name: CI
+
+on:
+  push:
+    branches:
+    - v8.8
+    - v8.9
+    - v8.10
+    - v8.11
+    - v8.12
+    - v8.13
+  pull_request:
+    branches:
+    - v8.8
+    - v8.9
+    - v8.10
+    - v8.11
+    - v8.12
+    - v8.13
```

Comment:
Actually this was added to the travis file to avoid double-building of pull requests that live in this particular repos, not sure this is still a problem with github actions.

Reply:
I think in fact PRs would get double-built, but I'm also not sure that's such a problem?  What's the bad thing that happens if they get double-built?

Reply:
Nothing bad, just a waste of runners (for Travis these were quite limited), so keeping the branch then sounds good.